Sources/Configuration/Sentry.xcconfig

```diff
@@ -5,3 +5,7 @@ PRODUCT_BUNDLE_IDENTIFIER = io.sentry.Sentry
 CURRENT_PROJECT_VERSION = 8.21.0
 
 MODULEMAP_FILE = $(SRCROOT)/Sources/Resources/Sentry.modulemap
+
+// This config is required so the test code can access the SentryPrivate module. Removing this setting
+// leads to an error: no such module '_SentryPrivate' when including the XCFramework with Carthage
+HEADER_SEARCH_PATHS = $(SRCROOT)/Sources/Sentry/include/**
```

Comment:
l: This setting is not only for test but also for the framework to find the module.

Reply:
Better now, @brustolin?